subprojects/parseq/src/main/java/com/linkedin/parseq/Task.java

```diff
@@ -893,6 +893,43 @@
     return withTimeout;
   }
 
+  /**
+   * Equivalent to {@code withDelay(null, time, unit)}.
+   * @see #withDelay(String, long, TimeUnit)
+   */
+  default Task<T> withDelay(final long time, final TimeUnit unit) {
+    return withDelay(null, time, unit);
+  }
+
+  /**
+   * Creates a new task whose execution is delayed.
+   * This task will complete with the exact same result as the delayed task, whether it is
+   * a success or failure.
+   * <blockquote><pre>
+   * final Task<Response> delayedRequest = HttpClient.get("http://google.com").task()
+   *     .withDelay(1, TimeUnit.SECONDS);
+   * </pre></blockquote>
+   *
+   * @param desc description of the delayed task. There is no need to put the delay value here because it will be
+   * automatically included, prepended to the description, if it is provided (e.g. withDelay 1s send request to Google).
```

Comment:
Please include the exact task description finally constructed, like how we did in `withTimeout` method.

Reply:
Updated the doc.